src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonFinderFeatures.kt

```diff
@@ -57,9 +57,15 @@ object DungeonFinderFeatures {
         "note",
         "§7§7Note: §f(?<note>.*)"
     )
+    /* 
+    REGEX-TESTS:
+    * REGEX-TEST: The Catacombs
+    * REGEX-TEST: Master Mode The Catacombs
+    * REGEX-TEST: MM Catacombs
```

Comment:
This test can't match that better. So either the pattern is wrong or the test is wrong